web/state/Roles.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+const ZERO = 0;
+const ERR_NO_ROLE_NAME = 1;
+const ERR_SPACE_IN_ROLE_LABEL = 2;
+const ERR_ROLE_ALREADY_EXISTS = 3;
+
+class Roles {
+  constructor (bdk, botName) {
```

Comment:
What would be the advantages of passing the `botName` in the constructor VS importing it from package.json or config.js? since the botName is static throughout the project.

Reply:
I preferred not importing from those files that could potentially move to different paths. It's already being imported in files that are creating the bdk so it's not much more hassle to pass it in as an arg.